app/uk/gov/hmrc/soletraderidentificationfrontend/repositories/JourneyConfigRepository.scala

```diff
@@ -80,6 +93,52 @@ object JourneyConfigRepository {
   val BusinessEntityKey = "businessEntity"
   val SoleTraderKey = "soleTrader"
   val IndividualKey = "individual"
+  val ContinueUrlKey = "continueUrl"
+  val PageConfigKey = "pageConfig"
+  val BusinessVerificationCheckKey = "businessVerificationCheck"
+  val RegimeKey = "regime"
+
+  val OptServiceNameKey = "optServiceName"
+  val DeskProServiceIdKey = "deskProServiceId"
+  val EnableSautrCheck = "enableSautrCheck"
+  val AccessibilityUrlKey = "accessibilityUrl"
+  val SignOutUrlKey = "signOutUrl"
+  val OptFullNamePageLabel = "optFullNamePageLabel"
+
 
-  implicit val journeyConfigMongoFormat: OFormat[JourneyConfig] = Json.format[JourneyConfig]
+  implicit val journeyConfigMongoFormat: OFormat[JourneyConfig] = new OFormat[JourneyConfig] {
```

Comment:
This doesn't need to be so complicated (see https://github.com/hmrc/incorporated-entity-identification-frontend/pull/147/files). However, temporarily you need to make "regime" optional to accommodate users who are in the middle of the journey when this code gets deployed.

The code below has to deal with optionality within the PageConfig data. Things are a little easier when a value inside JourneyConfig needs to be optional.

Reply:
done